### PR TITLE
chore(litmus-portal): Added user avatar selection modal

### PR DESCRIPTION
**Pull request by saswatamcode**
## Proposed changes
Added the user avatar selection modal which allows the user to select an avatar from eight different avatars as per the figma design. 

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [x] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Special notes for your reviewer:
Currently the avatar that is selected is stored as a local state in `UserDetails` as a string containing the path of the avatar svg. Will be integrated will backend later. This PR implements the changing functionality.

### CODE REVIEW

File: litmus-portal/frontend/src/views/Settings/UserManagementTab/CreateUser/ChooseAvatarModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+                    onClick={() => handleChange(av)}
+                  />
+                </Badge>
+              );
+            }
+            return (
+              <Avatar
+                key={av}
+                data-cy="avatar"
+                className={classes.av}
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/views/Settings/UserManagementTab/CreateUser/ChooseAvatarModal/index.tsx:85]: use better and more descriptive classnames

File: litmus-portal/frontend/src/views/Settings/UserManagementTab/CreateUser/ChooseAvatarModal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                alt="User"
+                src={av}
+                onClick={() => handleChange(av)}
+              />
+            );
+          })}
+        </div>
+      </div>
+      <ButtonFilled isDisabled={isDefault} isPrimary handleClick={handleSubmit}>
+        Choose an avatar
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/views/Settings/UserManagementTab/CreateUser/ChooseAvatarModal/index.tsx:136]: put in translation file

File: litmus-portal/frontend/src/views/Settings/UserManagementTab/CreateUser/UserDetails/index.tsx [hunk 3]
```diff
@@ -49,10 +59,18 @@ const UserDetails: React.FC<PersonalDetailsProps> = ({
...
               alt="User"
               className={classes.avatarBackground}
-              style={{ alignContent: 'right' }}
-            >
-              {initials}
-            </Avatar>
+              src={avatar}
+            />
+            <Button className={classes.edit} onClick={handleOpen}>
+              Edit Photo
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/views/Settings/UserManagementTab/CreateUser/UserDetails/index.tsx:65]: same

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]

**Review by arkajyotiMukherjee — APPROVED** [review]
LGTM cc: @S-ayanide

**Review by Saranya-jena — APPROVED** [review]
LGTM

**Pull request merged**